infrastructure/hub-and-spoke-peerings/README.md

```diff
@@ -0,0 +1,71 @@
+# Hub and Spoke via VPC Peering
+
+This sample creates a simple **Hub and Spoke** architecture, where the VPC network connects satellite locations (spokes) through a single intermediary location (hub) via [VPC Network Peering](https://cloud.google.com/vpc/docs/vpc-peering). 
+> **NOTE**: This example does not provide full mesh networking, if you need connectivity between spoke networks you will have to create a peering between every pair of spokes, or [use VPN](../hub-and-spoke-vpns). If you use additional peerings be aware of the [VPC Network Peering limits](https://cloud.google.com/vpc/docs/quota#vpc-peering).
+
+The benefits of this topology include:
+
+- Network/Security Admin manages Central Services Project (Hub).
+- Central services and tools deployed in Central Services Project (Hub) for use by all Service Projects (Spokes).
+- Network/Security Admin hands over spoke Projects to respective team who then have full autonomy.
+- Network/Security Admin monitors spoke projects for organization security posture compliance using tools like [Forseti](https://forsetisecurity.org/), [CSCC](https://cloud.google.com/security-command-center/) etc deployed in Central Services Project (Hub).
+- Spokes communicate with on-prem via VPC Network Peering to transit hub and then over Interconnect or VPN to on-premises (on-premises resources are not included in this sample for obvious reasons).
+- This is a decentralized architecture where each spoke project has autonomy to manage all their GCP compute and network resources.
+
+The purpose of this sample is showing how to wire different [Cloud Foundation Fabric](https://github.com/search?q=topic%3Acft-fabric+org%3Aterraform-google-modules&type=Repositories) modules to create **Hub and Spoke Peerings** network architectures, and as such it is meant to be used for prototyping, or to experiment with networking configurations. Additional best practices and security considerations need to be taken into account for real world usage (eg removal of default service accounts, disabling of external IPs, firewall design, etc).
```

Comment:
This is left-over from the old hub and spoke README, I would just remove it (use the hub and spoke vpn for reference).